INSTALL.rst

```diff
@@ -2,6 +2,41 @@
 Installation
 ============
 
+System prerequisites
+=====================
+
+This guide expects you to have installed is your system the following tools:
+
+* pip
+* virtualenv
+* virtualenvwrapper
+* npm > 3.0
+* postgresql + devel headers
+* libxml2 + devel headers
+* libxslt + devel headers
+* ImageMagick
+* redis
+* elasticsearch
+
+For example, for centos:
+
+.. code-block:: console
+
+    $ sudo yum install python-pip python-virtualenv python-virtualenvwrapper \
+        npm postgresql postgresql-devel lxml ImageMagick redis 
+    $ sudo npm -g install npm
+
+For elasticsearch you can find the installation instructions on the
+`elasticsearch install page`_, and, to run the development environment, you
+will need also add the following workarounds:
+
+.. code-block:: console
```

Comment:
Note that this is only in CentOS ? At least on MacOS it was not needed, might be good making it clear.